app/routes/task_routes.py

```diff
@@ -1 +1,64 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, Response
```

Comment:
`abort`, `make_response`, and `Response` are imported, but never accessed so they need to be removed.